### ISSUE TITLE
[FEATURE REQ] Add AddAzureMonitorLogExporter extension method on LoggerProviderBuilder 

### ISSUE THREAD

**Issue by joegoldman2** [Azure/azure-sdk-for-net#44464]
### Library name

Azure.Monitor.OpenTelemetry.Exporter

### Please describe the feature.

# Summary

Following https://github.com/open-telemetry/opentelemetry-dotnet/pull/5648, I think `Azure.Monitor.OpenTelemetry.Exporter` should provide a new extension method to configure the Azure Monitor log exporter on `LoggerProviderBuilder` (in the same way that `AddAzureMonitorTraceExporter` exists on `TracerProviderBuilder` and `AddAzureMonitorMetricExporter` exists for `MeterProviderBuilder`).

# API Proposal

```diff
namespace Azure.Monitor.OpenTelemetry.Exporter
{
    public static class AzureMonitorExporterExtensions
    {
+        public static LoggerProviderBuilder AddAzureMonitorLogExporter(
+            this LoggerProviderBuilder builder,
+            Action<AzureMonitorExporterOptions> configure = null,
+            TokenCredential credential = null)
    }
}
```

In a second phase, once this new API is available, perhaps `OpenTelemetryLoggerOptions.AddAzureMonitorLogExporter` should be deprecated.

cc @codeblanch, @lmolkova

**Comment by TimothyMothra** [comment]
Thanks for the reminder! This is already on our radar. We're waiting for the OTel 1.9.0 release which should be by the end of this week. 
Assuming schedules line up, this should make it into our July beta release. :)

**Comment by joegoldman2** [comment]
I guess this issue is fixed by #44617 and can be closed?
